webcompat/static/js/lib/models/issue.js

```diff
@@ -102,7 +91,7 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var regex = /\*\*Description\*\*: ([^*]+)/;
+    var regex = /<strong>Description<\/strong>: (.+?(?=<br>))/;
```

Comment:
This gives me butterfly. Regex and html are not good friends. :) Probably better to parse it as an html fragment and then extract what we want.

Reply:
okay :)
I also think it's a better idea. thanks for the tip